### PR TITLE
Bugfix/ios get optics metric fix

### PR DESCRIPTION
**Pull request by ExaneServerTeam**
This PR fixes the new layout of *show interfaces transceiver* on IOS where a new column "current" was added in the latest release

### CODE REVIEW

File: test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json [hunk 1]
```diff
@@ -1,669 +1 @@
...
...: {"instant": -3.8, "avg": -100.0, "min": -100.0, "max": -100.0}, "output_power": {"instant": -2.2, "avg": -100.0, "min": -100.0, "max": -100.0}, "laser_bias_current": {"instant": 5.5, "avg": 0.0, "min": 0.0, "max": 0.0}}}]}}, "TenGigabitEthernet7/27": {"physical_channels": {"channel": [{"index": 0, "state": {"input_power": {"instant": -2.7, "avg": -100.0, "min": -100.0, "max": -100.0}, "output_power": {"instant": -2.5, "avg": -100.0, "min": -100.0, "max": -100.0}, "laser_bias_current": {"instant": 7.9, "avg": 0.0, "min": 0.0, "max": 0.0}}}]}}, "TenGigabitEthernet7/29": {"physical_channels": {"channel": [{"index": 0, "state": {"input_power": {"instant": -2.1, "avg": -100.0, "min": -100.0, "max": -100.0}, "output_power": {"instant": -1.9, "avg": -100.0, "min": -100.0, "max": -100.0}, "laser_bias_current": {"instant": 8.2, "avg": 0.0, "min": 0.0, "max": 0.0}}}]}}, "FortyGigabitEthernet7/33": {"physical_channels": {"channel": [{"index": 0, "state": {"input_power": {"instant": -2.8, "avg": -100.0, "min": -100.0, "max": -100.0}, "output_power": {"instant": 0.9, "avg": -100.0, "min": -100.0, "max": -100.0}, "laser_bias_current": {"instant": 36.6, "avg": 0.0, "min": 0.0, "max": 0.0}}}]}}}
```
Comment (mirceaulinic) [inline test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json:1]: Could you re-format this using a tool like https://jsoneditoronline.org/  - maybe it'll help better understand what's changed. (same ask for the other JSON file)

Comment (ExaneServerTeam) [reply test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json]: @mirceaulinic the jsons have been updated as requested.

Comment (mirceaulinic) [reply test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json]: Thanks for the update @ExaneNetworkTeam (and sorry for getting back to this)! I see that the values for the `instant` have changed in many places for the existing test data, could you clarify why's that?

Comment (ExaneServerTeam) [reply test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json]: Hello,

This what the fix is for ;).
Basically on new IOS version a new column *Current* was added and no worries for the delay.
So in this case the output of ```show interfaces transciever``` was wrongly parsed.
(TxPower is Current value and Rx Power is Tx Power Value).

See sample cli output

*Old format*
```
                                 Optical   Optical
           Temperature  Voltage  Tx Power  Rx Power
Port       (Celsius)    (Volts)  (dBm)     (dBm)
---------  -----------  -------  --------  --------
Te1/0/1      34.6       3.29      -2.0      -3.5
Te2/0/1      34.3       3.28      N/A      -40.0

```
*New format*
```
            Temperature  Voltage  Current   Tx Power  Rx Power
Port        (Celsius)    (Volts)  (mA)      (dBm)     (dBm)
----------  -----------  -------  --------  --------  --------
Te3/2         26.5       3.30      38.1      -1.9     -26.8 --
Gi5/6         37.4       3.37      10.7      -4.9     -35.2 -
Gi5/12        40.0       3.35      10.6      -5.0      -4.6

```

Comment (mirceaulinic) [reply test/ios/mocked_data/test_get_optics/interface_shutdown_v2/expected_result.json]: Great, thank you for confirming! :smile:

**Pull request merged**